content/en/docs/topics/observing-spinkube.md

```diff
@@ -0,0 +1,153 @@
+---
+title: Observing SpinKube
```

Comment:
```suggestion
title: Observability in SpinKube

```

Reply:
I liked the former title because it describes an action the user can take, however I recognize that the original title doesn't necessarily reflect what the article walks through - the article discusses how to allow SpinApps to export telemetry data.

What if the title was changed to "Observing your applications"? We can omit "in SpinKube" - that is implied when reading the SpinKube documentation.

Reply:
Or maybe "Monitoring your app" to match the title of the topic guide "Connecting to your app"?

Reply:
I like "Monitoring your app".